examples/vision/3D_image_classification.py

```diff
@@ -0,0 +1,482 @@
+"""
+Title: 3D Image Classification from CT Scans
+Author: [Hasib Zunair](https://twitter.com/hasibzunair)
+Date created: 2020/09/23
+Last modified: 2020/09/23
+Description: Train a 3D convolutional neural network to predict presence of pneumonia.
+"""
+"""
+## Introduction
+
+This example will show the steps needed to build a 3D convolutional neural network (CNN)
+to predict the presence of viral pneumonia in computer tomography (CT) scans.
+
+2D CNNs are commonly used to process RGB images (3 channels). A 3D CNN is simply the 3D
+equivalent:
+it takes as input a 3D volume or a sequence of 2D frames (e.g. slices in a CT scan),
+3D CNNs are a powerful model for learning representations for volumetric data.
+
+## References
+
+- [A survey on Deep Learning Advances on Different 3D
+DataRepresentations](https://arxiv.org/pdf/1808.01462.pdf)
```

Comment:
Also, fix this link formatting, it won't render properly. Links should be on one line.

Reply:
Done!